.gitmodules

```diff
@@ -62,3 +62,6 @@
 	path = sources/poky
 	url = https://github.com/yoedistro/poky
 	branch = master
+[submodule "sources/meta-variscite-bsp"]
+	path = sources/meta-variscite-bsp
+	url = https://github.com/YoeDistro/meta-variscite-bsp.git
```

Comment:
I think we need to specify `branch = yoe/master` as well.

Reply:
thanks, fixed.